packages/paste-core/components/tabs/src/TabList.tsx

```diff
@@ -1,35 +1,40 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
 import {Box} from '@twilio-paste/box';
+import type {BoxProps} from '@twilio-paste/box';
 import {TabPrimitiveList} from '@twilio-paste/tabs-primitive';
 import {TabsContext} from './TabsContext';
-import {Variants} from './types';
+import type {Variants} from './types';
 
 export interface TabListProps {
   'aria-label': string;
   disabled?: boolean | undefined;
+  element?: BoxProps['element'];
+
   focusable?: boolean | undefined;
   children: React.ReactNode;
   variant?: Variants;
 }
 
-const HorizontalTabList: React.FC = ({children}) => (
+const HorizontalTabList: React.FC<{element?: BoxProps['element']}> = ({children, element}) => (
   <Box
     display="flex"
     borderBottomWidth="borderWidth10"
     borderBottomColor="colorBorderWeak"
     borderBottomStyle="solid"
+    element={`HORIZONTAL_${element}`}
```

Comment:
I now understand why HORIZONTAL/VERTICAL are prefixing customized element props in the tests. In this case I would do something like `element={element != null ? element : "HORIZONTAL_TAB_LIST"}`